utils/fix_xc7_carry.py

```diff
@@ -0,0 +1,595 @@
+""" Script for addressing CARRY4 output congestion in elaborated netlists.
+
+Usage:
+
+    python3 fix_carry.py < input-netlist-json > output-netlist-json
+
+Description:
+
```

Comment:
I just realized there is a missing description here.

The `C` and `COUTn` signals require **3** inputs to be generated, `S`, `DI` and `COUTn-1` (also called 'CIN').

Reply:
True, but this is not something `fix_xc7_carry` needs to care about, as that is all handled in the techmaps.